packages/mantine/src/components/table/table-date-range-picker/TableDateRangePicker.tsx

```diff
@@ -44,26 +44,25 @@ const defaultProps: Partial<TableDateRangePickerProps> = {
 };
 
 export const TableDateRangePicker = factory<TableDateRangePickerFactory>((props, ref) => {
```

Comment:
When you click outside the modal, should it close automatically?

![image](https://github.com/coveo/plasma/assets/966550/7fe04595-6afb-4f3f-a155-cd9ea822fd1c)

Reply:
I think yes, but it happens also on master branch, I will log a separate ticket for it